### PR TITLE
Update notebooks for QLV2 and PeripheralGroundAgent changes

### PR DESCRIPTION
**Pull request by mzielin**
Updated queries in two cosimulation notebooks to QLV2:
- examples/data_handling_cosimulation.ipynb
- examples/wildfire_cosimulation_game.ipnb

Updated one notebook with changes to PeripheralAgent kinematics:
- model_validation/attitude_dynamics/build_scenario.ipynb

### CODE REVIEW

File: examples/data_handling_cosimulation.ipynb [hunk 1]
```diff
@@ -142,6 +142,13 @@
     "### External State Blocks\n"
    ]
   },
+  {
```
Comment (mzielin) [inline examples/data_handling_cosimulation.ipynb:145]: Haven't used notebooks much, entirely sure where this metadata came from. Should it be removed?

Comment (bradsease) [reply examples/data_handling_cosimulation.ipynb:145]: Should be fine to leave in as long as the notebook works as expected

File: examples/data_handling_cosimulation.ipynb [hunk 2]
```diff
@@ -158,18 +165,16 @@
...
+    "    consumed=f'''(\n",
+    "        time,\n",
+    "        elapsedTime as Duration.s,\n",
+    "        block.{target_group.id}.activeTarget.(\n",
+    "            id, agentElevation as Angle.deg,\n",
+    "            lineOfSight, range as Distance.m,\n",
+    "            rangeRate as Speed.km/s\n",
+    "        )\n",
+    "    )''',\n",
+    "    produced=f'''block.{data_interface.id}.isActive''',\n",
```
Comment (bradsease) [inline examples/data_handling_cosimulation.ipynb:177]: Small style suggestion: can you put each element inside the `( )` on its own line?

File: examples/wildfire_cosimulation_game.ipynb [hunk 3]
```diff
@@ -411,34 +411,36 @@
...
-    "    'produced': [\n",
-    "        {\"ReactionWheel\": \"commandedTorqueMagnitude\"},\n",
-    "        {\"Thruster\": \"thrust\"}],\n",
+    "    'consumed':'''\n",
+    "        root!.attitude,\n",
+    "        root!.lvlhAxes,\n",
+    "        ReactionWheel.momentum,\n",
+    "        FuelTank.wetMass,\n",
+    "        Orbit.radiusApogee,\n",
+    "        Orbit.radiusPerigee\n",
```
Comment (bradsease) [inline examples/wildfire_cosimulation_game.ipynb:420]: Looks like `time` was dropped here?

File: examples/data_handling_cosimulation.ipynb (hunk 2 above)
Comment (mzielin) [reply examples/data_handling_cosimulation.ipynb]: Haha that's definitely what I was trying to do here, not sure how I did such a bad job!
Addressed in f30b97076e11a34d843325b61ff7c489de0c5808

File: examples/wildfire_cosimulation_game.ipynb (hunk 3 above)
Comment (mzielin) [reply examples/wildfire_cosimulation_game.ipynb]: Thanks for catching that, addressed in f30b97076e11a34d843325b61ff7c489de0c5808

File: examples/data_handling_cosimulation.ipynb [hunk 4]
```diff
@@ -182,13 +189,15 @@
...
    "source": [
     "power_state_block = scenario.SpontaneousExternalState.create(\n",
-    "    consumed=[{\"PowerProcessor\": {\"prev\": {\"outputPowers\": {\"index\": \"total\"}}}}],\n",
-    "    produced=[\n",
-    "        {f\"block.{modem.id}\": \"activeLoadState\"},\n",
-    "        {f\"block.{modem_load.id}\": \"isActive\"},\n",
-    "        {f\"block.{modem_load.id}\": \"powerConsumed\"},\n",
-    "    ],\n",
-    "    engineIndex=2,\n",
+    "    consumed='''(prev!(PowerProcessor.outputPowers["total"]),)''',\n",
```
Comment (bradsease) [inline examples/data_handling_cosimulation.ipynb:192]:
```suggestion
    "    consumed='''(prev!(PowerProcessor.outputPowers[\"total\"]),)''',\n",
```

File: examples/data_handling_cosimulation.ipynb [hunk 5]
```diff
@@ -158,19 +165,19 @@
...
+    "        block.{target_group.id}.activeTarget.(\n",
+    "            id,\n",
+    "            agentElevation as Angle.deg,\n",
+    "            lineOfSight,\n",
+    "            range as Distance.m,\n",
+    "            rangeRate as Speed.km/s\n",
+    "        )\n",
+    "    )''',\n",
+    "    produced=f'''(block.{data_interface.id}.isActive,)''',\n",
+    "    engine='CDH',\n",
```
Comment (bradsease) [inline examples/data_handling_cosimulation.ipynb:180]:
```suggestion
    "    consumed=f'''(\n",
    "        time,\n",
    "        elapsedTime as Duration.s,\n",
    "        block!("{target_group.id}").activeTarget.(\n",
    "            id,\n",
    "            agentElevation as Angle.deg,\n",
    "            lineOfSight,\n",
    "            range as Distance.m,\n",
    "            rangeRate as Speed.km/s\n",
    "        )\n",
    "    )''',\n",
    "    produced=f'''(block!("{data_interface.id}").isActive,)''',\n",
    "    engine='cdh',\n",
```

File: examples/data_handling_cosimulation.ipynb [hunk 6]
```diff
@@ -182,13 +189,15 @@
...
-    "    engineIndex=2,\n",
+    "    consumed='''(prev!(PowerProcessor.outputPowers[\"total\"]),)''',\n",
+    "    produced=f'''\n",
+    "        (\n",
+    "            block.{modem.id}.activeLoadState,\n",
+    "            block.{modem_load.id}.isActive,\n",
+    "            block.{modem_load.id}.powerConsumed\n",
+    "        )\n",
+    "    ''',\n",
+    "    engine='CDH',\n",
```
Comment (bradsease) [inline examples/data_handling_cosimulation.ipynb:200]:
```suggestion
    "    consumed='''(prev!(PowerProcessor.outputPowers[\"total\"]),)''',\n",
    "    produced=f'''\n",
    "        (\n",
    "            block!("{modem.id}").activeLoadState,\n",
    "            block!("{modem_load.id}").isActive,\n",
    "            block!("{modem_load.id}").powerConsumed\n",
    "        )\n",
    "    ''',\n",
    "    engine='power',\n",
```

File: examples/data_handling_cosimulation.ipynb [hunk 7]
```diff
@@ -158,19 +165,19 @@
...
+    "        block!("{target_group.id}").activeTarget.(\n",
+    "            id,\n",
+    "            agentElevation as Angle.deg,\n",
+    "            lineOfSight,\n",
+    "            range as Distance.m,\n",
+    "            rangeRate as Speed.km/s\n",
+    "        )\n",
+    "    )''',\n",
+    "    produced=f'''(block!("{data_interface.id}").isActive,)''',\n",
+    "    engine='cdh',\n",
```
Comment (bradsease) [inline examples/data_handling_cosimulation.ipynb:180]:
```suggestion
    "    consumed=f'''(\n",
    "        time,\n",
    "        elapsedTime as Duration.s,\n",
    "        block!(\"{target_group.id}\").activeTarget.(\n",
    "            id,\n",
    "            agentElevation as Angle.deg,\n",
    "            lineOfSight,\n",
    "            range as Distance.m,\n",
    "            rangeRate as Speed.km/s\n",
    "        )\n",
    "    )''',\n",
    "    produced=f'''(block!(\"{data_interface.id}\").isActive,)''',\n",
    "    engine='cdh',\n",
```

File: examples/data_handling_cosimulation.ipynb [hunk 8]
```diff
@@ -182,13 +189,15 @@
...
-    "        {f\"block.{modem_load.id}\": \"isActive\"},\n",
-    "        {f\"block.{modem_load.id}\": \"powerConsumed\"},\n",
-    "    ],\n",
-    "    engineIndex=2,\n",
+    "    consumed='''(prev!(PowerProcessor.outputPowers[\"total\"]),)''',\n",
+    "    produced=f'''\n",
+    "        (\n",
+    "            block!("{modem.id}").activeLoadState,\n",
+    "            block!("{modem_load.id}").isActive,\n",
+    "            block!("{modem_load.id}").powerConsumed\n",
```
Comment (bradsease) [inline examples/data_handling_cosimulation.ipynb:197]:
```suggestion
    "            block!(\"{modem.id}\").activeLoadState,\n",
    "            block!(\"{modem_load.id}\").isActive,\n",
    "            block!(\"{modem_load.id}\").powerConsumed\n",
```

File: examples/wildfire_cosimulation_game.ipynb [hunk 9]
```diff
@@ -411,34 +411,37 @@
...
+    "    ''',\n",
+    "    'produced':'''\n",
+    "        ReactionWeel.commandedTorqueMagnitude,\n",
+    "        Thruster.thrust\n",
+    "    ''',\n",
```
Comment (bradsease) [inline examples/wildfire_cosimulation_game.ipynb:426]:
```suggestion
    "    'consumed': '''\n",
    "        (\n",
    "            time,\n",
    "            root!.attitude,\n",
    "            root!.lvlhAxes,\n",
    "            ReactionWheel.momentum,\n",
    "            FuelTank.wetMass,\n",
    "            Orbit.radiusApogee,\n",
    "            Orbit.radiusPerigee\n",
    "        )\n",
    "    ''',\n",
    "    'produced':'''\n",
    "        (\n",
    "            ReactionWheel.commandedTorqueMagnitude,\n",
    "            Thruster.thrust\n",
    "        )\n",
    "    ''',\n",
```

File: examples/wildfire_cosimulation_game.ipynb [hunk 10]
```diff
@@ -411,34 +411,37 @@
...
     "    'engineIndex': 0,\n",
     "    'agents': [WILDFIRE_ID],\n",
     "}\n",
     ")\n",
     "GNC_STATE_ID = gnc_state_block.id\n",
     "cdh_state_block = scenario.SpontaneousExternalState.create(**{\n",
-    "    'produced': [{\"CombinationalLogic\": \"activeSubroutine\"}],\n",
+    "    'produced':'''CombinationalLogic.activeSubroutine''',\n",
     "    'engineIndex': 1,\n",
```
Comment (bradsease) [inline examples/wildfire_cosimulation_game.ipynb:434]:
```suggestion
    "    'consumed': '()',\n",
    "    'produced': '''(CombinationalLogic.activeSubroutine,)''',\n",
    "    'engine': 'cdh',\n",
```

File: examples/wildfire_cosimulation_game.ipynb [hunk 11]
```diff
@@ -411,34 +411,37 @@
...
-    "    'consumed': [\n",
-    "        {\"Battery\": \"soc\"},\n",
-    "        {\"PowerProcessor\": \"outputPowers\"},\n",
-    "        {\"SolarArray\": \"utilization\"}],\n",
+    "    'consumed':'''\n",
+    "        Battery.soc,\n",
+    "        PowerProcessor.outputPowers,\n",
+    "        SolarArray.utilization\n",
+    "    ''',\n",
     "    'engineIndex': 2,\n",
```
Comment (bradsease) [inline examples/wildfire_cosimulation_game.ipynb:445]:
```suggestion
    "    'consumed': '''\n",
    "        (\n",
    "            Battery.soc,\n",
    "            PowerProcessor.outputPowers,\n",
    "            SolarArray.utilization\n",
    "        )\n",
    "    ''',\n",
    "    'produced': '()',\n",
    "    'engine': 'power',\n",
```

File: examples/wildfire_cosimulation_game.ipynb [hunk 12]
```diff
@@ -411,35 +411,46 @@
...
+    "            Orbit.radiusPerigee\n",
+    "        )\n",
+    "    ''',\n",
+    "    'produced':'''\n",
+    "        (\n",
+    "            ReactionWheel.commandedTorqueMagnitude,\n",
+    "            Thruster.thrust\n",
+    "        )\n",
+    "    ''',\n",
     "    'engineIndex': 0,\n",
```
Comment (bradsease) [inline examples/wildfire_cosimulation_game.ipynb:431]:
```suggestion
    "    'engine': 'gnc,\n",
```

File: examples/wildfire_cosimulation_game.ipynb [hunk 13]
```diff
@@ -411,35 +411,46 @@
...
+    "            Orbit.radiusPerigee\n",
+    "        )\n",
+    "    ''',\n",
+    "    'produced':'''\n",
+    "        (\n",
+    "            ReactionWheel.commandedTorqueMagnitude,\n",
+    "            Thruster.thrust\n",
+    "        )\n",
+    "    ''',\n",
+    "    'engine': 'gnc,\n",
```
Comment (bradsease) [inline examples/wildfire_cosimulation_game.ipynb:431]:
```suggestion
    "    'engine': 'gnc',\n",
```

**Pull request merged**